NEWS.md

```diff
@@ -3,6 +3,7 @@
 * Apparmor userspace is vendored inside the snapd snap
 * Added a default-configure hook that exposes gadget default configuration
   options to snaps during first install before services are started
+* Allow install from initrd to speed up the initial installation
```

Comment:
shouldn't this say more about current limitations?

Reply:
Good point, I added a note on the (current) limitations and added also a few more of the 2.60 featues